optimum/intel/ipex/modeling_base.py

```diff
@@ -60,7 +64,7 @@
 logger = logging.getLogger(__name__)
 
 
-_IPEX_SUPPORT_MODEL_TYPES = ("llama",)
+_IPEX_SUPPORT_MODEL_TYPES = ("llama", "bert", "vit")
```

Comment:
could you add tests for both architectures ?

Reply:
The Bert and Vit models are already in the tests, and they will automatically go into the patching path. Please give me more instructions, like what kind of tests you need. Thanks!